### PR TITLE
fix: fix GuildBankRedirector pattern [skip ci]

### PR DESCRIPTION
**Pull request by cmkohnen**
Also since, these messages appear under `RecipientType.GUILD` remove the check for only INFO messages.

### CODE REVIEW

**Review by kristofbolyai — APPROVED** [review]
It doesn't seem like removing the recipient type check has any performance impact. However, I'll do revert this if anything significant pops up (that I couldn't see during my testing).

File: common/src/main/java/com/wynntils/features/redirects/ChatRedirectFeature.java [hunk 1]
```diff
@@ -133,8 +132,6 @@ private void register(Redirector redirector) {
 
     @SubscribeEvent(priority = EventPriority.HIGH)
     public void onChatMessage(ChatMessageReceivedEvent e) {
-        if (e.getRecipientType() != RecipientType.INFO) return;
```
Comment (magicus) [inline common/src/main/java/com/wynntils/features/redirects/ChatRedirectFeature.java:136]: So why did you not change it to GUILD instead?

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/redirects/ChatRedirectFeature.java:136]: Because this event applies all chat redirects. We are still in debt regarding this one.

Comment (magicus) [reply common/src/main/java/com/wynntils/features/redirects/ChatRedirectFeature.java:136]: Ah, now I see. Previously we only redirected info messages, now we also have a guild one. 

But let me rephrase it:

So why did you not change it to INFO or GUILD instead?

That this is dropped means we will check for redirects in all global chat messages etc.

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/redirects/ChatRedirectFeature.java:136]: Yes, we can do that. I've not had that idea. If I remember next time I'm developing, I'll PR it.
